lib/robots/dor_repo/gis_delivery/load_geoserver.rb

```diff
@@ -233,7 +232,7 @@ def create_raster(connection, workspace_name)
               logger.debug "load-geoserver: #{bare_druid} loaded style #{raster_style}"
               if style_exists.nil?
                 logger.debug "load-geoserver: #{bare_druid} saving new style #{raster_style}"
-                style.create(style_name: raster_style)
+                style.create(style_name: raster_style, filename: nil)
                 style.update(style_name: raster_style, filename: nil, payload: sldtxt)
```

Comment:
Note: writing the tests uncovered the fact that this method is missing a required parameter.  Here is the definition in the gem: https://github.com/samvera-labs/geoserver-publish/blob/main/lib/geoserver/publish/style.rb#L28-L32

Reply:
I guess we don't hit this code path very often (at all?) or else it should have triggered a robot error and/or an HB alert

Reply:
This reminds me that I noticed we have quite a few DRUID specific tiles in Production GeoServer but not in Stage. I wonder if this stopped working at some point? Perhaps it's worth a discussion after standup with @kimdurante unless this has already come up?